rest_framework_jwt/views.py

```diff
@@ -124,7 +154,7 @@ def get(self, request):
         return Response({}, status=status.HTTP_200_OK)
```

Comment:
you can send device id in response to login for them to store which should be send everytime in headers.
Or they can send you permanent token in delete call.